controllers/argocd/notifications/configmap.go

```diff
@@ -27,7 +27,6 @@ func (nr *NotificationsReconciler) reconcileConfigMap() error {
 
 	if err != nil {
 		nr.Logger.Error(err, "reconcileConfigMap: failed to request configMap", "name", desiredConfigMap.Name, "namespace", desiredConfigMap.Namespace)
-		nr.Logger.V(1).Info("reconcileConfigMap: one or more mutations could not be applied")
```

Comment:
why did we get rid of this?

Reply:
@jaideepr97 I didn't get why request configMap is "one or more mutation could not be applied". Can you explain when do we need to log this and why?